scraper/src/documentation_spider.py

```diff
@@ -103,13 +109,12 @@ def parse_from_sitemap(self, response):
         if (not self.force_sitemap_urls_crawling) and (not self.is_rules_compliant(response)):
             print("\033[94m> Ignored from sitemap:\033[0m " + response.url)
         else:
-            self.add_records(response)
-
-        return self.parse(response)
+            self.add_records(response, True)
+        # We don't return self.parse(response) in order to avoid crawling those web page
 
     def parse_from_start_url(self, response):
         if self.is_rules_compliant(response):
-            self.add_records(response)
+            self.add_records(response, False)
```

Comment:
You should have a look into python named argument. It allows to be more readable especially when passing booleans